extensions/brew/src/brew.ts

```diff
@@ -113,11 +117,7 @@ export function brewPath(suffix: string): string {
 }
 
 export function brewExecutable(): string {
-  if (preferences.customBrewPath && preferences.customBrewPath.length > 0) {
-    return preferences.customBrewPath;
-  } else {
-    return path_join(brewPrefix, "bin/brew");
-  }
+  return customBrewPath ?? path_join(brewPrefix, "bin/brew");
```

Comment:
Just use `return brewPath("bin/brew");` here. The `brewPrefix` will have the custom path if necessary.